### PR TITLE
Release/auto

### PR DESCRIPTION
**Pull request by chylli-deriv**

### CODE REVIEW

File: dist.ini [hunk 1]
```diff
@@ -1,70 +1,12 @@
...
-[CheckPrereqsIndexed]
-[CheckExtraTests]
-[VersionFromModule]
-[ReversionOnRelease]
-[NextRelease]
-[OurPkgVersion]
-[InstallGuide]
-[MetaJSON]
-[InsertExample]
+author  = bianry.com <binary@cpan.org>
```
Comment (nihal-deriv) [inline dist.ini]:
```suggestion
author  = binary.com <BINARY@cpan.org>
```

File: dist.ini [hunk 2]
```diff
@@ -1,70 +1,12 @@
...
-[CheckPrereqsIndexed]
-[CheckExtraTests]
-[VersionFromModule]
-[ReversionOnRelease]
-[NextRelease]
-[OurPkgVersion]
-[InstallGuide]
-[MetaJSON]
-[InsertExample]
+author  = binary.com <binary@cpan.org>
```
Comment (nihal-deriv) [inline dist.ini]: i have noticed that the author email for all the other repos have been BINARY@cpan.org.

Comment (chylli-deriv) [reply dist.ini]: You are right. Thanks

Comment (chylli-deriv) [inline dist.ini]:
```suggestion
author  = binary.com <BINARY@cpan.org>
```

**Pull request merged**